generators/guestbinary/index.js

```diff
@@ -150,6 +149,17 @@ module.exports = generators.Base.extend({
             var serviceTypeName = this.props.serviceName + 'Type';
             var appTypeName = this.projName + 'Type';
             var pkgDir = this.isAddNewService == false ? path.join(this.projName, this.projName) : this.projName;
+            var is_Windows = (process.platform == 'win32');
+            var is_Linux = (process.platform == 'linux');
+            var is_mac = (process.platform == 'darwin');
```

Comment:
These variables are unused?

Reply:
Linux and Mac are not used. Removed them.